tests/unit/test_pyproject_dev_dependencies.py

```
@@ -0,0 +1,210 @@
+
+    def test_add_dev_dependencies_to_existing_pep621_config(
+        self, sample_pyproject_toml: str
+    ):
+        """Test adding dev dependencies to existing PEP 621 pyproject.toml."""
+        # This test should fail initially since we haven't implemented the functionality
+        with tempfile.NamedTemporaryFile(mode="w", suffix=".toml", delete=False) as f:
+            f.write(sample_pyproject_toml)
+            f.flush()
+
+            # This should add the dev dependencies
+            from src.config.pyproject_manager import PyprojectManager
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)